assets/src/edit-story/app/story/effects/useLoadStory.js

```diff
@@ -41,37 +41,46 @@ function useLoadStory({ storyId, shouldLoad, restore }) {
         const {
           title: { raw: title },
           status,
-          author,
           slug,
           date_gmt,
           modified,
           excerpt: { raw: excerpt },
           link,
           story_data: storyDataRaw,
-          featured_media_url: featuredMediaUrl,
+          // todo: get publisher_logo_url image dimensions for prepublish checklist
           publisher_logo_url: publisherLogoUrl,
           permalink_template: permalinkTemplate,
           style_presets: stylePresets,
           password,
-          _embedded: embedded,
+          _embedded: embedded = {},
         } = post;
         const date = `${date_gmt}Z`;
 
+        let author = {
+          id: 0,
+          name: '',
+        };
+
+        if ('author' in embedded) {
+          author = {
+            id: embedded.author[0].id,
+            name: embedded.author[0].name,
+          };
+        }
+
         let featuredMedia = {
           id: 0,
           height: 0,
           width: 0,
-          url: featuredMediaUrl,
+          url: '',
         };
 
-        if (embedded && embedded['wp:featuredmedia']) {
-          const featuredMediaData = embedded['wp:featuredmedia'][0];
-
+        if ('wp:featuredmedia' in embedded) {
           featuredMedia = {
-            id: featuredMediaData?.id,
-            height: featuredMediaData?.media_details?.height,
-            width: featuredMediaData?.media_details?.width,
-            url: featuredMediaUrl,
+            id: embedded['wp:featuredmedia'][0].id,
+            height: embedded['wp:featuredmedia'][0].media_details?.height,
+            width: embedded['wp:featuredmedia'][0].media_details?.width,
+            url: embedded['wp:featuredmedia'][0].source_url,
```

Comment:
@swissspidy Shouldn't we use the poster size here?

Reply:
I created another ticket for this - https://github.com/google/web-stories-wp/issues/5228

Reply:
It doesn't matter too much for output as we override it in PHP. Let's handle this in #5228 as well